src/test/java/com/clickhouse/kafka/connect/sink/ClickHouseSinkTaskWithSchemaTest.java

```diff
@@ -148,6 +150,7 @@ public Collection<SinkRecord> createArrayType(String topic, int partition) {
             Struct value_struct = new Struct(NESTED_SCHEMA)
                     .put("off16", (short)n)
                     .put("arr", arrayTmp)
+                    .put("arr_empty", arrayEmpty)
```

Comment:
We could simplify the test and add some empty records array and, in some of their array with values.
Just less code and tests could be more real.